okonomiyaki/file_formats/pep425.py

```diff
@@ -55,3 +61,21 @@ def pep425_tag(self):
 
     def __str__(self):
         return "{0.abbreviated_implementation}{0.major}{0.minor}".format(self)
+
+
+def _abbreviated_implementation():
+    """Return abbreviated implementation name."""
+    if sys.platform.startswith('java'):
+        pyimpl = 'jy'
+    elif sys.platform == 'cli':
+        pyimpl = 'ip'
+    elif hasattr(sys, 'pypy_version_info'):
+        pyimpl = 'pp'
+    else:
```

Comment:
I think this should be an explicit condition for cpython, with an `else: raise UnknownImplementation()`.